contrib/grpc-cmake/CMakeLists.txt

```diff
@@ -1331,8 +940,6 @@ add_library(grpc++
   ${GRPC_SOURCE_DIR}/src/cpp/codegen/codegen_init.cc
 )
 
-target_compile_options(grpc++ PUBLIC -w)
```

Comment:
It was harmful command because it disabled all warnings for all targets which linked with `grpc++`.
So I have removed it.